### DISCUSSION TITLE
如何将Bilibili的视频嵌入网页呢？

### DISCUSSION THREAD

**Discussion by wangjueszu** [requarks/wiki#5624] [Help / Questions]
从https://www.bilibili.com/上获得的嵌入代码是这个。我直接放在页面里面没有效果

<iframe src="//player.bilibili.com/player.html?aid=769531438&bvid=BV1Er4y1x7mf&cid=732685131&page=1" scrolling="no" border="0" frameborder="no" framespacing="0" allowfullscreen="true"> </iframe>

**Comment by cdn0x12** [comment]
Currently you'll need to use the markdown editor or raw HTML editor to insert you r video iframe. And please make sure you have enabled `Allow iframes` in settings -> rendering -> html to html -> security.
